mods/beds/functions.lua

```diff
@@ -224,6 +239,27 @@ minetest.register_on_leaveplayer(function(player)
 	end
 end)
 
+minetest.register_on_dieplayer(function(player)
+	local name = player:get_player_name()
+	local in_bed = beds.player
+	local pos = player:get_pos()
+	local yaw = get_look_yaw(pos)
```

Comment:
Deprecated, use `get_look_horizontal` instead.

Reply:
It's a helper function from line `13`.